docker/buildkit.go

```diff
@@ -54,16 +69,32 @@ func BuildDockerImageWithBuildKit(
 
 	eg, ctx := errgroup.WithContext(ctx)
 
-	export := client.ExportEntry{
-		Type: "image",
-		Attrs: map[string]string{
-			"name": strings.Join(tags, ","),
-		},
-	}
-	if push {
-		export.Attrs["push"] = "true"
+	exports := make([]client.ExportEntry, 1)
+
+	if tarExport != "" {
+		exports[0] = client.ExportEntry{
```

Comment:
I realize this only works with single platform.

Is it necessary to add a check ensuring only one platform? Default is "linux/arm64,linux/amd64"

Reply:
great catch! added an error to be more informative rather than the buildkit build failing.